{"url": "https://api.github.com/repos/tensorflow/tensorflow/issues/comments/241988897", "html_url": "https://github.com/tensorflow/tensorflow/issues/3459#issuecomment-241988897", "issue_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/3459", "id": 241988897, "node_id": "MDEyOklzc3VlQ29tbWVudDI0MTk4ODg5Nw==", "user": {"login": "untom", "id": 3627551, "node_id": "MDQ6VXNlcjM2Mjc1NTE=", "avatar_url": "https://avatars2.githubusercontent.com/u/3627551?v=4", "gravatar_id": "", "url": "https://api.github.com/users/untom", "html_url": "https://github.com/untom", "followers_url": "https://api.github.com/users/untom/followers", "following_url": "https://api.github.com/users/untom/following{/other_user}", "gists_url": "https://api.github.com/users/untom/gists{/gist_id}", "starred_url": "https://api.github.com/users/untom/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/untom/subscriptions", "organizations_url": "https://api.github.com/users/untom/orgs", "repos_url": "https://api.github.com/users/untom/repos", "events_url": "https://api.github.com/users/untom/events{/privacy}", "received_events_url": "https://api.github.com/users/untom/received_events", "type": "User", "site_admin": false}, "created_at": "2016-08-24T08:09:18Z", "updated_at": "2016-08-24T08:09:18Z", "author_association": "NONE", "body_html": "<p>I can confirm that the behavior seems fixed with the current nightlies</p>", "body_text": "I can confirm that the behavior seems fixed with the current nightlies", "body": "I can confirm that the behavior seems fixed with the current nightlies\n"}